GoogleForCreators/web-stories-wp
# Block: Fix issues with rearranging stories in Firefox

## Context

From https://github.com/GoogleForCreators/web-stories-wp/pull/12649:

On firefox, the rearrange overlay did not allow me to move the stories by clicking on the cover image !! I could onlu rearrange when I dragged the title area ...

Plugin Version: 1.27.0
WordPress Version: 6.1
Operating System:
Browser: Firefox

## Summary

Updates CSS for `web-stories-story-preview-card__overlay` making it relative vs absolute 

See:

Testing in Chrome and Firefox
👉  https://www.youtube.com/watch?v=EXu6hDamKBw


## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Add WP stories block 
2. Select block
3. Press rearrange
4. Drag and drop the cover image 
5. View the block to ensure the new order is correct


## Reviews

### Does this PR have a security-related impact?

No



### Does this PR change what data or activity we track or use?

No



### Does this PR have a legal-related impact?

No



## Checklist




---



Fixes #12703
